journalbeat/plan.sh

```diff
@@ -0,0 +1,35 @@
+pkg_name=journalbeat
+pkg_origin=core
+pkg_version="6.6.0"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_deps=(core/glibc core/systemd)
+pkg_build_deps=(core/go core/git core/make core/gcc core/systemd core/patchelf)
+pkg_bin_dirs=(bin)
+pkg_svc_user=root
+pkg_svc_group=root
+pkg_description="Journalbeat is a lightweight journald log shipper for Elasticsearch."
+pkg_upstream_url="https://github.com/elastic/beats/tree/master/journalbeat"
+
+do_download() {
```

Comment:
This function is automatically a no-op if `pkg_source` isn't defined.   It's safe to remove this.

Reply:
removed